torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2092,6 +2092,55 @@ def sample_inputs_like_fns(self, device, dtype, requires_grad, **kwargs):
 
     return tuple(samples)
 
+def sample_inputs_randint_like(self, device, dtype, requires_grad, **kwargs):
+    samples = []
+    low = 2
+    high = 10
+
+    for sample in sample_inputs_like_fns(self, device, dtype, requires_grad, **kwargs):
+        # With high
+        samples.append(SampleInput(sample.input, args=(high,) + sample.args, kwargs=sample.kwargs))
```

Comment:
Reusing the same tensor across SampleInputs is going to be a problem. These probably need something like `.detach().clone().requires_grad_(requires_grad)` (which can be put in a helper because it's very verbose). Also I suggest rebasing this to pick up the latest OpInfo tests which are more likely to fail if SampleInputs share tensors.